src/main/java/com/dougnoel/sentinel/elements/WindowsElement.java

```diff
@@ -182,38 +182,22 @@ public Element click() {
 	 * @param yOffset int the pixels below the top edge of the element
 	 * @return Color the color at the specified pixel offset in the element.
 	 */
-	public Color getColor(int xOffset, int yOffset) {
-		Robot robot;
-		try {
-			robot = new Robot();
-		} catch (AWTException awte) {
-			throw new com.dougnoel.sentinel.exceptions.IOException(awte);
-		}
-
-		// Grab the pixel coordinates of the rectangle of the element, relative to the top-left corner of the screen. Positive X = right. Positive Y = down. 
```

Comment:
I'm not saying that this needs to change, but I noticed in my click position work that the 'official' W3 standard is no longer to use the top left, and instead the center of the element. You should see what people suggest as best practice.

Note #4: https://www.w3.org/TR/webdriver/#wheel-actions

Reply:
Need to have a discussion on this as I believe we initially avoided middle since text color could be captured if we went dead center.